frontend/public/components/utils/dropdown.jsx

```diff
@@ -642,47 +642,104 @@ Dropdown.propTypes = {
   dataTest: PropTypes.string,
 };
 
-class ActionsMenuDropdown_ extends DropdownMixin {
-  render() {
-    const { actions, title = undefined, t } = this.props;
-    const onClick = (event, option) => {
-      event.preventDefault();
+const ActionsMenuDropdown_ = (props) => {
+  const { t } = useTranslation();
+  const [active, setActive] = React.useState(!!props.active);
+
+  const dropdownElement = React.useRef();
+
+  const show = () => {
+    setActive(true);
+  };
+
+  const hide = (e) => {
+    e && e.stopPropagation();
```

Comment:
We can use optional chaining to simplify this:
```suggestion
    e?.stopPropagation();
```